internal/indexer/indexer.go

```diff
@@ -0,0 +1,131 @@
+package indexer
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"sync"
+
+	"github.com/Taraxa-project/taraxa-indexer/internal/chain"
+	"github.com/Taraxa-project/taraxa-indexer/internal/storage"
+	"github.com/Taraxa-project/taraxa-indexer/models"
+)
+
+type Indexer struct {
+	client  *chain.WsClient
+	storage *storage.Storage
+}
+
+func NewIndexer(url string, storage *storage.Storage) (i *Indexer, err error) {
+	i = new(Indexer)
+	i.storage = storage
+	i.client, err = chain.NewWsClient(url)
+
+	return
+}
+
+func (indexer *Indexer) Index() {
+	for i := uint64(1050); i < 1055; {
```

Comment:
@kstdl what means those numbers?
- 1050
- 1055

Reply:
Thats block numbers. So this cycle just to test some blocks indexing, will be removed later